CMakeLists.txt

```diff
@@ -136,9 +136,20 @@ message("Webmin: ${SSG_PRODUCT_WEBMIN}")
 
 message(" ")
 
+include(SSGCommon)
+
+# Targets 'validate' and 'zipfile' need to be added before any product because
+# they will receive dependencies from products added
+
 add_custom_target(validate)
```

Comment:
Since things in SSGCommon rely on custom target `validate` being present I would expect it to be defined before the include is actually included. I also remember that I had to change the order in the past to avoid issues. Maybe these don't manifest anymore.